dist/api_data.py

```diff
@@ -1021,6 +1021,15 @@ def __ge__(self, other):
 wiredtiger_open_live_restore_configuration = [
     Config('live_restore', '', r'''Live restore configuration options. These options control the
     behavior of WiredTiger when live restoring from a backup.''', type='category', subconfig = [
+        Config('debug', '', r'''
+            configure debug specific behavior on Live Restore. Generally only used for internal
+            testing purposes.''',
+            type='category', subconfig=[
+            Config('fill_holes_on_close', 'false', r'''
```

Comment:
Do we plan on removing this when the background thread is done? Or will we keep a config available so we can do some specific kind of testing? 

Also the undoc=True flag should be set on the `debug` `Config`. Also no need to capitalize 'Live Restore'

Reply:
Fixed the undoc, typo in [032235b](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/032235bdbfca37396e0b27efdba71fb13308b7b3). If we remove this is being discussed in https://github.com/wiredtiger/wiredtiger/pull/11332#discussion_r1884620676

Reply:
The undoc=True should go the level above.

Reply:
[d9981b3](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/d9981b3d59256a6c1cbf7942abc58d808e0c212b)